qa/workunits/rados/test-upgrade-v12.0.0.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -ex
+
+parallel=1
+[ "$1" = "--serial" ] && parallel=0
+
+color=""
+[ -t 1 ] && color="--gtest_color=yes"
+
+function cleanup() {
+    pkill -P $$ || true
+}
+trap cleanup EXIT ERR HUP INT QUIT
+
+pids=""
+for f in \
+    'api_aio --gtest_filter=-LibRadosAio.RacingRemovePP:*WriteSame*:*CmpExt*:*RoundTrip3*:*RoundTripPP3*:*Quota*' \
```

Comment:
I don't think we need a separate test-upgrade-v12.0.0.sh yet

Reply:
@jdurgin which one do we use then ?